components/ipfs/features.cc

```diff
@@ -6,11 +6,19 @@
 #include "brave/components/ipfs/features.h"
 
 #include "base/feature_list.h"
+#include "brave/components/ipfs/buildflags/buildflags.h"
```

Comment:
I think we need to add buildflags into deps in `brave/components/ipfs/BUILD.gn` for this.

Reply:
Oops I assumed it was already there. Rebased and force pushed the fix here: 98d2e875da796cd754ffdd22643c983293dc6ff3